src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/restrict/ClientAccessControlService.java

```diff
@@ -16,53 +13,16 @@
 @Service
 public class ClientAccessControlService {
 
-    private final LaunchDarklyFeatureFlagProvider launchDarklyFeatureFlagProvider;
     private final ServiceAuthTokenValidator serviceAuthTokenValidator;
-    private final List<String> privilegedAccessClients;
     private final List<String> exclusiveAccessClients;
 
     @Autowired
     public ClientAccessControlService(ServiceAuthTokenValidator serviceAuthTokenValidator,
-                                      LaunchDarklyFeatureFlagProvider launchDarklyFeatureFlagProvider,
-                                      @Value("${config.privilegedAccessClients}") List<String> privilegedAccessClients,
                                       @Value("${config.exclusiveAccessClients}") List<String> exclusiveAccessClients) {
         this.serviceAuthTokenValidator = serviceAuthTokenValidator;
-        this.launchDarklyFeatureFlagProvider = launchDarklyFeatureFlagProvider;
-        this.privilegedAccessClients = privilegedAccessClients;
         this.exclusiveAccessClients = exclusiveAccessClients;
     }
 
-
-    /**
-     * Extracts client id from service authorization token and returns if client is whitelisted as privilegedServices.
-     * Note: This feature is sitting behind feature flag.
-     *
-     * @param serviceAuthToken      the service authorization token.
-     * @param accessControlResponse the access control response containing userId.
-     * @return whether a client has been whitelisted in config.privilegedAccessClients property.
-     */
-    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
-    public boolean hasPrivilegedAccess(String serviceAuthToken, AccessControlResponse accessControlResponse) {
```

Comment:
Not sure if this is correct.
I think the requirement was to remove the feature flag rather than the functionality of checking that the client has permission to make this request.

So I would expect that the if/else logic on here based on a feature flag is removed but not the whole method.

Reply:
Method added back. Only feature flag check removed.